app/src/main/java/com/framgia/soundclound/util/BindingUtil.java

```diff
@@ -14,6 +14,7 @@
 import android.widget.TextView;
 
 import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.target.BitmapImageViewTarget;
```

Comment:
ERROR: Unused import: com.bumptech.glide.request.target.BitmapImageViewTarget. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports